Automattic/wp-calypso
# A4A: Remove the out-dated Site dashboard empty state.

When a user has no connected sites, the empty state displays two buttons: one to download the A4A plugin, and the other to connect a site via Jetpack (see the screenshot). From a user's perspective, these two options are inferior to showing the new Site Selector component.

This PR resolves this issue by not showing the empty state and simply displaying the site table with the create site button.

| Before | After |
|--------|--------|
| <img width="1453" alt="Screenshot 2024-08-12 at 6 46 09 PM" src="https://github.com/user-attachments/assets/9e6b150e-1539-486f-ad96-a0e6e077ccf1"> | <img width="1456" alt="Screenshot 2024-08-12 at 6 45 55 PM" src="https://github.com/user-attachments/assets/e13e00ee-a584-42be-86dc-f3c0e5b2459e"> | 

Closes https://github.com/Automattic/automattic-for-agencies-dev/issues/816

## Proposed Changes

* Remove the Site dashboard's empty state.

## Why are these changes being made?

* This PR encourages a superior method of adding new or existing sites to A4A.

## Testing Instructions

* Create a new Agency with no sites.
* Use the A4A live link and go to the `/sites` page.
* Confirm that you no longer see the empty message.

## Pre-merge Checklist
